app/views/master_files/hls_manifest.m3u8.erb

```diff
@@ -14,7 +14,8 @@ Unless required by applicable law or agreed to in writing, software distributed
 ---  END LICENSE_HEADER BLOCK  ---
 %>
 #EXTM3U
+#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="subs",LANGUAGE="en",NAME="English",AUTOSELECT=YES,URI="<%= caption_manifest_master_file_path %>"
```

Comment:
Final implementation could have a setting to specify a default caption language to avoid having to hardcode it as english. If implemented, would be good to eventually circle back and allow editors to define caption language, as well as provide an option to provide multiple caption files.